packages/node_modules/@cerebral/mobx-state-tree/README.md

```diff
@@ -44,48 +42,41 @@ import { types } from 'mobx-state-tree'
 import { set } from 'cerebral/operators'
 import { state, props } from 'cerebral/tags'
 
-const Box = types.model('Box', {
-  id: types.identifier(),
-  name: '',
-  x: 0,
-  y: 0,
-  get width() {
-    return this.name.length * 15
-  }
-})
-
 export default Module({
   // Define your model as an object,
-  // it will be converted to a types.model
+  // it will be converted to a mobx model
   model: {
-    boxes: types.map(Box),
-    currentBoxId: types.maybe(types.string)
+    name: types.string,
   },
   // Define the initial state as normal
   state: {
-    boxes: {},
-    currentBoxId: null
+    name: ''
   },
-  // You can define views as normal. It is encouraged to do so directly
-  // on the module, though nested models can of course also have views. Note
-  // that views are only to GET state, not to change it. Meaning that in actions
-  // do not use views with for example state.set etc., only state.get
-  views(state) {
-    return {
-      currentBox() {
-        return state.boxes[state.currentBoxId]
-      }
+  // Define getters
+  // store.upper // "BAR"
```

Comment:
@christianalfoni the method in the comment is different from the function name ("upper" -> "upperName").